apps/arm/main.cpp

```diff
@@ -180,7 +187,25 @@ static mbed_error_status_t setPIDTuningMode(CANMsg &msg) {
 
 // Configure PID parameters
 static mbed_error_status_t setPIDParameter(CANMsg &msg) {
-    return MBED_SUCCESS; // TODO
+    // ASSUME PAYLOAD IS 32 BITS LONG
+    int32_t payload(0);
+    msg >> payload;
+    // The actuatorID is made of the 4 LSBs of the payload
+    uint8_t actuatorID = (payload & 15);
+    // 5th LSB indicates whether we are tuning velocity, if false, we are tuning position
```

Comment:
I don't think you should make that assumption. rather explicitly check and have an error default case

Reply:
will do

Reply:
wait i didnt realize it was a single bit

Reply:
i thought ur comment was related to the assumption that the DLC of the CAN packet is 32 bits. is there value in checking that the received CAN packet has a 32 bit payload?

Reply:
no.